analyses/UT_cd_2000/03_sim_UT_cd_2000.R

```diff
@@ -0,0 +1,35 @@
+###############################################################################
+# Simulate plans for `UT_cd_2000`
+# © ALARM Project, July 2025
+###############################################################################
+
+# Run the simulation -----
+cli_process_start("Running simulations for {.pkg UT_cd_2000}")
+
+set.seed(2000)
+plans <- redist_smc(map, nsims = 2e3, runs = 5, counties = pseudo_county)
+plans <- plans %>%
+  group_by(chain) %>%
+  filter(as.integer(draw) < min(as.integer(draw)) + 1000) %>% # thin samples
+  ungroup()
+plans <- match_numbers(plans, "cd_2000")
+
+cli_process_done()
+cli_process_start("Saving {.cls redist_plans} object")
+
+# Output the redist_map object. Do not edit this path.
+write_rds(plans, here("data-out/UT_2000/UT_cd_2000_plans.rds"), compress = "xz")
+cli_process_done()
+
+# Compute summary statistics -----
+cli_process_start("Computing summary statistics for {.pkg UT_cd_2000}")
+
+plans <- add_summary_stats(plans, map)
+
+# Output the summary statistics. Do not edit this path.
+save_summary_stats(plans, "data-out/UT_2000/UT_cd_2000_stats.csv")
+
+cli_process_done()
+
+summary(plans)
```

Comment:
Can you place this back inside the `if (interactive)`? The goal of that is to ensure that anyone sourcing the files won't have extra files created, just the outputs.